File: abp/src/Wallee.Boc.Vote.Application.Contracts/Appraisements/AppraisementDto.cs
using System;
using Volo.Abp.Application.Dtos;
using Volo.Abp.Domain.Entities;

namespace Wallee.Boc.Vote.Appraisements
{
    public class AppraisementDto : AuditedEntityDto<Guid>, IHasConcurrencyStamp
    {
        public string Name { get; set; } = null!;
        public string Description { get; set; } = null!;
        public EvaluationCategory Category { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public string ConcurrencyStamp { get; set; } = null!;
    }
}

File: vote_vite_admin/src/openapi/models/AppraisementDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { AppraisementCategory } from './AppraisementCategory';

export type AppraisementDto = {
    id?: string;
    name?: string | null;
    description?: string | null;
    category?: AppraisementCategory;
    start?: string;
    end?: string;
    concurrencyStamp?: string | null;
};
